open-api/rest-catalog-open-api.yaml

```diff
@@ -1966,6 +2021,17 @@ components:
               "GET /v1/{prefix}/namespaces/{namespace}/tables/{table}",
               "GET /v1/{prefix}/namespaces/{namespace}/views/{view}"
             ]
+        idempotencyKeyLifetime:
+          type: string
+          format: duration
+          description: >
+            Client reuse window for an Idempotency-Key (ISO-8601 duration, e.g., PT30M, PT24H).
+            Interpreted as the maximum time from the first submission using a key to the last retry
+            during which a client may reuse that key. Servers MUST accept replays for at least this
```

Comment:
> Servers MUST accept replays for at least

nit:  I don't think we should say `MUST` here.  Servers may reject a request for any number of reasons and I don't think the spec should try an force behavior here.  This would be better as a `SHOULD`.

Reply:
Fixed. Thanks